src/_includes/components/footer.njk

```diff
@@ -15,8 +15,11 @@
           <label for="newsletter-email">Email address</label>
           <div class="field--connected-btn">
             <input id="newsletter-email" type="email" name="email" 
-              placeholder="name@example.com" autocapitalize="off" required />
-            <input type="hidden" name="tags" value="Web Sign up"/>
+              placeholder="name@example.com" autocapitalize="off" 
+              pattern="^[^@]+@[^@]+\.[^@]+$"
```

Comment:
Since we're already using `type=email` we get standard email validation https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email#Validation 

I'm not sure that we need the `pattern` here. I don't think it adds additional needed checks.

Reply:
Fixed!